Provide a way to add my custom notes in the agenda view when in log modeIndex: devel/config/elfiles/org/lisp/org-agenda.el
===================================================================
Index: devel/config/elfiles/org/lisp/org-agenda.el
===================================================================
--- devel.orig/config/elfiles/org/lisp/org-agenda.el	2013-11-18 09:38:56.728157159 +0100
+++ devel/config/elfiles/org/lisp/org-agenda.el	2013-11-18 09:38:56.724157175 +0100
@@ -5788,6 +5788,22 @@
 	 (parts
 	  (delq nil
 		(list
+		 (let (
+			(note
+			 (cdr
+			  (assoc 'note org-log-note-headings)
+			  )
+			 )
+			)
+		    (save-match-data
+		      (if (string-match "^\\(.+\\)%t$" note)
+			  (format "- %s"
+			   (match-string-no-properties 1 note)
+			   )
+			nil
+			)
+		      )
+		    )
 		 (if (memq 'closed items) (concat "\\<" org-closed-string))
 		 (if (memq 'clock items) (concat "\\<" org-clock-string))
 		 (if (memq 'state items) "- State \"\\([a-zA-Z0-9]+\\)\".*?"))))
